app/columns.tsx

```diff
@@ -255,43 +256,44 @@ export const experimentColumns: ColumnDef<ExperimentAndBranchInfo>[] = [
           && props.row.original.surface !== 'spotlight') {
           return ( <></> );
       }
+    }
+  }, {
+    accessorKey: "previewLink",
+    header: "",
+    cell: (props: any) => {
+      if (props.row.original.previewLink == undefined) {
+        return ( <div/> );
+      }
 
       // unless / until we expose via UITour (or MAKE_LINKABLE?)
       const copyPreviewLink = () => {
         return navigator.clipboard.writeText(props.row.original.previewLink);
       }
 
-      return (
-        copyPreviewLink ?
-        <TooltipProvider>
-          <Tooltip>
-            <TooltipTrigger asChild>
-                <Button
-                  className={
-                    buttonVariants({
-                        variant: "secondary",
-                        size: "sm",
-                        className: "active:bg-slate-500 font-normal border  border-slate-700"
-                    })
-                  }
-                  onClick={copyPreviewLink}>
-                <Copy className="me-2" />
-                Copy Preview URL
-              </Button>
-              </TooltipTrigger>
-              <TooltipContent>
-                <p>After clicking to copy, paste in URL bar for message preview</p>
-              </TooltipContent>
-            </Tooltip>
-          </TooltipProvider>
-
-        :
-        <a
-          className={buttonVariants({ variant: "outline", size: "sm" })}
-          href={props.row.original.previewLink}
-          target="_blank">
-          Preview
-        </a> );
+    return (
+      copyPreviewLink ?
```

Comment:
The other side of the fork (in the ASRouter table) is actually more up-to-date re-styling.  Can you move that code into a PreviewLinkButton component in its own file and then call it from both there and here?

Reply:
Can do!